WorkcraftCore/src/org/workcraft/gui/graph/tools/AbstractModelConverter.java

```diff
@@ -44,8 +44,9 @@ public TDstModel getDstModel() {
         return dstModel;
     }
 
+    @SuppressWarnings("PMD.UnusedPrivateMethod") // PMD gets confused for some reason
     private void putSrcToDstNode(VisualNode srcNode, VisualNode dstNode) {
-        if ((srcNode != null) && (dstNode != null)) {
```

Comment:
I think this was just a drive-by cleanup, so I'll undo it.

Either way, it should be clear that `!=` has precedence over `&&`. I've never seen anyone use parenthesis in this context. Moreover, the expression makes no sense if you reverse the precedence.